aten/src/ATen/native/native_functions.yaml

```diff
@@ -3398,6 +3398,10 @@
 
 - func: fbgemm_pack_gemm_matrix_fp16(Tensor input) -> Tensor
 
+- func: wrapped_linear_prepack(Tensor weight, Tensor weight_scale, Tensor weight_zero_point, Tensor bias) -> Tensor
+
+- func: wrapped_quantized_linear_prepacked(Tensor input, Tensor input_scale, Tensor input_zero_point, Tensor packed_weight, Tensor output_scale, Tensor output_zero_point, int out_channel) -> Tensor
```

Comment:
You don't need these -- these generate aten::wrapped_linear_prepack ops. It looks like you're only defining _quantized::wrapped_linear_prepack